install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8';
+
+ALTER TABLE `PREFIX_search_word` MODIFY `word` VARCHAR(30) NOT NULL;
```

Comment:
Missing new line at the end of the script :D

Reply:
done !